<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QWidget" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>860</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <widget class="QGroupBox" name="groupChannelSelect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>671</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Channel Selection</string>
   </property>
   <widget class="QCheckBox" name="checkBoxCH01">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH01</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH02">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH02</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH03">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH03</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH04">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH04</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH05">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH05</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH06">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH06</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH07">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH07</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH08">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH08</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH09">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH09</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH10">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH10</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH11</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCH12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CH12</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCHALL">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ALL</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupParametersCFD_ZERO">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>CFD_ZERO Calibration Parameters</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="aDCMIPLabel">
       <property name="text">
        <string>ADC/MIP:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditADCpMIP">
       <property name="text">
        <string>16</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="attenuatorStepsLabel">
       <property name="text">
        <string>Initial Attenuator Steps:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditInitAttenSteps">
       <property name="text">
        <string>7200</string>
       </property>
       <property name="maxLength">
        <number>5</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupChannelStatus">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Calibration Status</string>
   </property>
   <widget class="QCheckBox" name="statusCH01">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH01</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH02">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH02</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH03">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH03</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH04">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH04</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH05">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH05</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH06">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH06</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH07">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH07</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH08">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH08</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH09">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH09</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH10">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="statusCH12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {background-color:red;}</string>
    </property>
    <property name="text">
     <string>CH12</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupCommands">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Commands</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAbort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupCalibrationTypes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Calibration Types</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>102</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QRadioButton" name="radioButtonTimeAlign">
       <property name="text">
        <string>Time Alignment</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonADC_DELAY">
       <property name="text">
        <string>ADC_DELAY</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonCFD_ZERO">
       <property name="text">
        <string>CFD_ZERO</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>911</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabALL">
    <attribute name="title">
     <string>ALL</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEditALL">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>911</width>
       <height>581</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabCH01">
    <attribute name="title">
     <string>CH01</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH02">
    <attribute name="title">
     <string>CH02</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH03">
    <attribute name="title">
     <string>CH03</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH04">
    <attribute name="title">
     <string>CH04</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH05">
    <attribute name="title">
     <string>CH05</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH06">
    <attribute name="title">
     <string>CH06</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH07">
    <attribute name="title">
     <string>CH07</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH08">
    <attribute name="title">
     <string>CH08</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH09">
    <attribute name="title">
     <string>CH09</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH10">
    <attribute name="title">
     <string>CH10</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH11">
    <attribute name="title">
     <string>CH11</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCH12">
    <attribute name="title">
     <string>CH12</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonRestoreTimeDelays">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore TimeDelays</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
